dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -162,8 +170,8 @@ export default function CollectCardPaymentScreen() {
         },
       ],
     });
-    const paymentMethods = ['card_present'];
-    if (enableInterac) {
+    const paymentMethods = enabledPaymentMethodTypes;
```

Comment:
it might not useful to declare another local variable here, given all modification should also sync back to `enabledPaymentMethodTypes ` via `setEnabledPaymentMethodTypes `. And if we don't sync it correctly, there might be logic error now or later.

Reply:
since we have an external button for `interac_present`, i think its fine to keep the separate, as toggling the button shouldn't affect the list

Reply:
Got it